yarn.lock

```diff
@@ -1105,6 +1112,15 @@
     "@babel/parser" "^7.25.9"
     "@babel/types" "^7.25.9"
```

Comment:
You should be able to have all of these use the newer version, rather than installing two copies.  Same for a bunch of the other packages you are changing.